sycl/include/CL/sycl/detail/sycl_fe_intrins.hpp

```diff
@@ -21,6 +21,16 @@ SYCL_EXTERNAL T __sycl_getSpecConstantValue(const char *ID);
 template <typename T>
 SYCL_EXTERNAL T __sycl_getCompositeSpecConstantValue(const char *ID);
 
+// The intrinsics below are used to enable support of specialization constants
+// 2020. This feature required post-link to handle more parameters.
```

Comment:
Thanks! One more nit

```suggestion
// The intrinsics below are used to implement support SYCL2020 specialization constants.
// SYCL2020 version requires more parameters compared to the initial version.
```